doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml

```diff
@@ -23,17 +23,15 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: raycluster-complete-logs
 spec:
-  rayVersion: '2.0.0'
+  rayVersion: '2.3.0'
```

Comment:
Is there a reason we updated this? do we expect this example to work with `2.3.0` and older versions of ray in general?

Reply:
I am not sure about the policy here. In KubeRay repository, we will use the latest release of Ray in every sample YAMl file. 

> do we expect this example to work with 2.3.0 and older versions of ray in general?

Yes